Gemfile

```diff
@@ -18,3 +18,5 @@ gem 'therubyrhino', platforms: :jruby
 gem 'jruby-openssl', platforms: :jruby
 
 gem 'sqlite3', platforms: :ruby
+
+gem "rubocop", require: false
```

Comment:
Please note that according to this documentation, HoundCI is using version 0.54 of rubocop:
http://help.houndci.com/configuration/rubocop

Maybe it is the reason that HoundCI is still pending (so it crashed because of an incompatible configuration?)